spec/types_from_serializers/__snapshots__/interfaces_ComposerSerializer.snap

```diff
@@ -2,10 +2,9 @@
 //
 // DO NOT MODIFY: This file was automatically generated by TypesFromSerializers.
 
-export default interface Composer {
+export interface Composer {
```

Comment:
The "standard" mode should not be affected by this PR, as it's a breaking change, and I'd like to ship namespace support in a minor release.

Let's keep the `default` export when `namespace` is `nil`.